bazel.sh

```diff
@@ -19,21 +19,22 @@
 # This script will clean, build, and run tests against each installation of Xcode available on the
 # machine using bazel.
 #
-# Arguments:
-#   1. bazel action (build or test, usually)
-#   2. BUILD target.
-#   3. Minimum Xcode version. E.g. "8" or "8.2.1"
+# Ordered arguments:
+#   1. action:  bazel ACTION. E.g. "build" or "test".
+#   2. target:  Bazel build target. E.g. "//path/to/target:Target"
+#
+# Named arguments:
+#   -m|--min-xcode-version <version>: Minimum Xcode version. E.g. "8" or "8.2.1"
+#   -v|--verbose:                     Generates verbose output on local runs.
+#                                     Does not affect kokoro runs.
 #
 # Example usage:
-#   bazel.sh build //:CatalogByConvention
-#   bazel.sh test //:CatalogByConventionTests
+#   bazel.sh build //:CatalogByConvention --xcode-version 8.2
+#   bazel.sh test //:CatalogByConventionTests -v
 
 # Fail on any error.
 set -e
 
-# Display commands to stderr.
-set -x
-
 script_version="v3.2.0"
 echo "bazel_build_and_test version $script_version"
```

Comment:
Change this to echo "$0 version $script_version"? Or maybe something like

```bash
echo "$(basename $0) version $script_version"
```

Reply:
Done.